plugins/module_utils/puppet.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+import os
+
+from ansible_collections.community.general.plugins.module_utils.cmd_runner import CmdRunner, cmd_runner_fmt
+
+
+_puppet_path_prefix = ["/opt/puppetlabs/bin"]
```

Comment:
I would use upper case for file-level constants:
```suggestion
_PUPPET_PATH_PREFIX = ["/opt/puppetlabs/bin"]
```

Reply:
done